docs/reference/elasticsearch/rest-apis/searching-with-query-rules.md

```diff
@@ -15,8 +15,8 @@ $$$query-rules$$$
 * A user-entered query string
 * Personalized metadata about users (e.g. country, language, etc)
 * A particular topic
-* A referring site
-* etc.
+* A referring site and so on
```

Comment:
```suggestion
* A referring site
```

personal opinion: I don't think we need either, because "For example" makes it clear this isn't an exhaustive list